h/auth/__init__.py

```diff
@@ -22,24 +41,33 @@
 
 log = logging.getLogger(__name__)
 
-PROXY_POLICY = RemoteUserAuthenticationPolicy(environ_key='HTTP_X_FORWARDED_USER',
-                                              callback=groupfinder)
-TICKET_POLICY = pyramid_authsanity.AuthServicePolicy()
-
+# Configure policy for authentication against API services.
+# The API policy makes use of two "sub-policies",
+# Token and AuthClient. The API policy in turn is used by the
+# app-wide policy.
 TOKEN_POLICY = TokenAuthenticationPolicy(callback=groupfinder)
 AUTH_CLIENT_POLICY = AuthClientPolicy()
-
 API_POLICY = APIAuthenticationPolicy(user_policy=TOKEN_POLICY,
                                      client_policy=AUTH_CLIENT_POLICY)
 
+
+# The default policy for the entire app combines the API policy
+# (for API requests) with a ticket policy (for other requests)
+TICKET_POLICY = pyramid_authsanity.AuthServicePolicy()
 DEFAULT_POLICY = AuthenticationPolicy(api_policy=API_POLICY,
                                       fallback_policy=TICKET_POLICY)
+
+
+# If proxy auth is enabled, PROXY_POLICY is used instead of DEFAULT_POLICY
+PROXY_FALLBACK_POLICY = RemoteUserAuthenticationPolicy(environ_key='HTTP_X_FORWARDED_USER',
+                                                       callback=groupfinder)
+PROXY_POLICY = AuthenticationPolicy(api_policy=API_POLICY,
+                                    fallback_policy=PROXY_FALLBACK_POLICY)
+
 WEBSOCKET_POLICY = TOKEN_POLICY
 
 
 def includeme(config):
-    global DEFAULT_POLICY
```

Comment:
AFAICT, this is redundant and unused...no grep hits for this anywhere else in the app